src/app/SDLApp.cpp

```diff
@@ -93,6 +98,11 @@ SDLApp::SDLApp(const SDLApp::Settings &settings) :
 
 SDLApp::~SDLApp()
 {
+    // Destroy imgui
```

Comment:
Shouldn't these be in a separate commit?

Reply:
yup